pennylane/tape/qscript.py

```diff
@@ -834,25 +834,67 @@ def numeric_type(self) -> Union[type, tuple[type, ...]]:
     # Transforms: QuantumScript to QuantumScript
     # ========================================================
 
-    def copy(self, copy_operations: bool = False) -> "QuantumScript":
-        """Returns a shallow copy of the quantum script.
+    def copy(self, copy_operations: bool = False, **update) -> "QuantumScript":
+        """Returns a copy of the quantum script. If `update` was passed, the updated attributes are
+        modified, otherwise, all attributes match the original tape. The copy is a shallow copy if
+        `copy_operations` and `update` are both `False`.
```

Comment:
When will `update` be `False`?

Reply:
It won't anymore, I missed that part in the docs update!